receiver/mongodbatlasreceiver/metadata.yaml

```diff
@@ -623,18 +615,16 @@ metrics:
       value_type: double
   mongodbatlas.disk.partition.utilization.max:
     enabled: true
-    description: Disk partition utilization (%)
+    description: The percentage of time during which requests are being issued to and serviced by the partition.
     extended_documentation: MongoDB Metrics MAX_DISK_PARTITION_UTILIZATION
     unit: 1
-    attributes: [disk_status]
     gauge:
       value_type: double
   mongodbatlas.disk.partition.utilization.average:
     enabled: true
-    description: Disk partition utilization (%)
+    description: The maximum percentage of time during which requests are being issued to and serviced by the partition.
```

Comment:
This description says "maximum", but the metric name is "average". Was it switched with the other metric?

Reply:
Yup, copy paste mixup, good catch.